dcicutils/qa_utils.py

```diff
@@ -0,0 +1,20 @@
+"""
+qa_utils: Tools for use in quality assurance testing.
+"""
+
+from .misc_utils import PRINT
+
+
+def mock_not_called(name):
```

Comment:
Why not `assert not mock.called, "mock called where not expected"` ? Is the point that you preserve args/kwargs?

Reply:
Short answer is: personal style. But the style isn't entirely arbitrary in choice. I find those mock properties to be easy to make typos on. I've misspelled them and found it had ill effect. e.g., suppose you do "if mock.callled" or something like that, what ends up happening is it returns a magic mock object that counts as true. So I tend to not rush to use those things unless the thing is hard to mock otherwise. Plus you don't have to change the structure of the code if you decide adjust what the function does, whereas if you want to go from "not mock.called" to doing something slightly different, you have to write not just the changed functionality but change the test.